src/test/spec/unified_runner/test_file.rs

```diff
@@ -268,6 +268,7 @@ pub struct CollectionOrDatabaseOptions {
     #[serde(rename = "readPreference")]
     pub selection_criteria: Option<SelectionCriteria>,
     pub write_concern: Option<WriteConcern>,
+    pub comment: Option<Bson>,
```

Comment:
Why was this change needed? The `comment` field shouldn't be specified at the db/coll level

Reply:
That was an oversight on my part. I confused the `CreateCollectionOptions` with `CollectionOptions` and ended up propagating that error up. I'll fix this and push again.